pkg/pairtree/pairtree.go

```diff
@@ -39,6 +39,7 @@ const (
 	verDir    = "pairtree_version0_1"
 	PtPrefix  = "pt://"
 	tar       = ".tgz"
+	ptVerSpec = "This directory conforms to Pairtree Version 0.1. Updated spec: http://www.cdlib.org/inside/diglib/pairtree/pairtreespec.html "
```

Comment:
Reminds me I should ask CDL again about whether these docs will get put back up.